.github/workflows/e2e-tests.yml

```diff
@@ -11,13 +11,13 @@ jobs:
     name: Tests
     runs-on: ubuntu-latest
     steps:
+      - name: Check out the code
+        uses: actions/checkout@v3
+
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19
-
-      - name: Check out the code
-        uses: actions/checkout@v3
+          go-version-file: go.mod
```

Comment:
If we checkout the code first, we can use the go version within the go.mod instead in order to make it the single source of truth for the desired go version. Only needing to change it in 1 place from now on.